seatunnel-connectors-v2/connector-common/src/main/java/org/apache/seatunnel/connectors/seatunnel/common/schema/SeatunnelSchema.java

```diff
@@ -0,0 +1,204 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.seatunnel.connectors.seatunnel.common.schema;
+
+import org.apache.seatunnel.api.table.type.ArrayType;
+import org.apache.seatunnel.api.table.type.BasicType;
+import org.apache.seatunnel.api.table.type.DecimalType;
+import org.apache.seatunnel.api.table.type.LocalTimeType;
+import org.apache.seatunnel.api.table.type.MapType;
+import org.apache.seatunnel.api.table.type.SeaTunnelDataType;
+import org.apache.seatunnel.api.table.type.SeaTunnelRowType;
+import org.apache.seatunnel.api.table.type.SqlType;
+import org.apache.seatunnel.common.config.CheckConfigUtil;
+import org.apache.seatunnel.common.config.CheckResult;
+
+import org.apache.seatunnel.shade.com.typesafe.config.Config;
+import org.apache.seatunnel.shade.com.typesafe.config.ConfigValue;
+
+import java.util.Map;
+
+public class SeatunnelSchema {
+    private static final String FIELD_KEY = "fields";
+    private static final String FORMAT_KEY = "format";
```

Comment:
I think the schema only expresses the structure of the table, and the format should be defined and supported by the connector.

Reply:
Use `seatunnel-formats`?

Reply:
For mysql and oracle connectors, users can also want to define schema, in which case format is invalid.
 I think format should be another feature (in fact, there is currently `json format` in the `seatunnel-format-json` module);

Reply:
So your guys think format information should be taken out in this class and abstract format information out to another feature?

Reply:
The format feature already exists, but it is not yet enabled on the entire process.

Reply:
OK, I will only keep schema information in it.

Reply:
@ashulin @Hisoka-X @CalvinKirs pls review thx.

Reply:
By the way, now we didn't contain formation information in our config, so whether config should be changed as the following:

```hcon
schema {
    map = "map<string, string>"
    array = "array<tinyint>"
    string = string
    boolean = boolean
    tinyint = tinyint
    smallint = smallint
    int = int
    bigint = bigint
    float = float
    double = double
    decimal = "decimal(30, 8)"
    null = "null"
    bytes = bytes
    date = date
    time = time
    timestamp = timestamp
}
```

Waiting for your guys advice.